llvm/test/CodeGen/AMDGPU/sgpr-spills-empty-block.ll

```diff
@@ -0,0 +1,473 @@
+; CHECK-NEXT:    v_readlane_b32 s7, v0, 7
+; CHECK-NEXT:  .LBB0_88: ; %Flow213
+; CHECK-NEXT:    s_mov_b64 s[0:1], s[22:23]
+; CHECK-NEXT:  .LBB0_89: ; %.sink.split
+; CHECK-NEXT:    s_load_dword s2, s[2:3], 0x0
+; CHECK-NEXT:    v_mov_b32_e32 v1, 0
+; CHECK-NEXT:    ; kill: killed $vgpr0
+; CHECK-NEXT:    s_waitcnt lgkmcnt(0)
+; CHECK-NEXT:    v_mov_b32_e32 v2, s2
+; CHECK-NEXT:    global_store_dword v1, v2, s[0:1]
+; CHECK-NEXT:    s_endpgm
+  switch i32 %47, label %.sink.split [
+    i32 75, label %63
+    i32 1, label %49
+    i32 2, label %50
+    i32 3, label %51
+    i32 4, label %52
+    i32 5, label %53
+    i32 6, label %54
+    i32 7, label %55
+    i32 82, label %70
+    i32 81, label %69
+    i32 80, label %68
+    i32 79, label %67
+    i32 78, label %66
+    i32 13, label %56
+    i32 14, label %57
+    i32 15, label %58
+    i32 16, label %59
+    i32 17, label %60
+    i32 18, label %61
+    i32 19, label %62
+    i32 77, label %65
+    i32 0, label %64
+  ]
+
+49:                                               ; preds = %48
+  br label %.sink.split
+
+50:                                               ; preds = %48
+  br label %.sink.split
+
+51:                                               ; preds = %48
+  br label %.sink.split
+
+52:                                               ; preds = %48
+  br label %.sink.split
+
+53:                                               ; preds = %48
+  br label %.sink.split
+
+54:                                               ; preds = %48
+  br label %.sink.split
+
+55:                                               ; preds = %48
+  br label %.sink.split
+
+56:                                               ; preds = %48
+  br label %.sink.split
+
+57:                                               ; preds = %48
+  br label %.sink.split
+
+58:                                               ; preds = %48
+  br label %.sink.split
+
+59:                                               ; preds = %48
+  br label %.sink.split
+
+60:                                               ; preds = %48
+  br label %.sink.split
+
+61:                                               ; preds = %48
+  br label %.sink.split
+
+62:                                               ; preds = %48
+  br label %.sink.split
+
+63:                                               ; preds = %48
+  br label %.sink.split
+
+64:                                               ; preds = %48
+  br label %.sink.split
+
+65:                                               ; preds = %48
+  br label %.sink.split
+
+66:                                               ; preds = %48
+  br label %.sink.split
+
+67:                                               ; preds = %48
+  br label %.sink.split
+
+68:                                               ; preds = %48
+  br label %.sink.split
+
+69:                                               ; preds = %48
+  br label %.sink.split
+
+70:                                               ; preds = %48
```

Comment:
Use named values

Reply:
Still trying to reduce the mir version - with this ll input, the mir is about 1700 lines long.